solutions/src/main/java/ru/sberbank/school/task02/Solution.java

```diff
@@ -0,0 +1,33 @@
+package ru.sberbank.school.task02;
+
+import java.math.BigDecimal;
+
+import ru.sberbank.school.task02.exception.FxConversionException;
+import ru.sberbank.school.task02.util.ClientOperation;
+import ru.sberbank.school.task02.util.ExternalQuotesServiceDemo;
+import ru.sberbank.school.task02.util.Symbol;
+
+public class Solution {
+    public static void main(String[] args) {
+
+        CalculatorFactory calculatorFactory = new CalculatorFactory();
+        // получаем наш валютный калькулятор
+        FxConversionService calculator =
+                calculatorFactory.getFxConversionService(new ExternalQuotesServiceDemo());
+
+        try {
+            BigDecimal price = calculator.convert(ClientOperation.BUY,
+                    Symbol.USD_RUB, new BigDecimal(100));
+            System.out.println(price);
+
+        }
```

Comment:
&#39;}&#39; at column 9 should be on the same line as the next part of a multi-block statement (one that directly contains multiple blocks: if/else-if/else, do/while or try/catch/finally).

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-372985677>*